src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectOptionsProcessor.cs

```diff
@@ -24,16 +26,18 @@ internal class VisualStudioProjectOptionsProcessor : IDisposable
         private readonly object _gate = new object();
         private string _commandLine = "";
         private CommandLineArguments _commandLineArgumentsForCommandLine;
-        private string _explicitRuleSetFilePath;
-        private IReferenceCountedDisposable<ICacheEntry<string, IRuleSetFile>> _ruleSetFile = null;
+        private string? _explicitRuleSetFilePath;
+        private IReferenceCountedDisposable<ICacheEntry<string, IRuleSetFile>>? _ruleSetFile = null;
```

Comment:
Can you not touch this file.  I'm editing it in a current PR (and NRT enabling it myself).  Thanks!

Reply:
I *could* revert it but it was only by annotating this did I catch some annotation issues in other files. Which other PR is this? I'm happy to defer the merge.

This PR also has another problem in that it's depending on some 16.6 nullability behavior changes but right now our build is still using an older Roslyn, so I can't actually merge this due to CI. So this can't go in right away anyways.

Reply:
My PR is in.  So go ahead with this :)

Reply:
@CyrusNajmabadi I've rebased this PR to merge with your changes. Our approaches were, uh, quite similar. Only remaining change on my side was just dropping the "opt" suffix which we've been doing as a part of annotations.